include/eve/concept/abi.hpp

```diff
@@ -0,0 +1,56 @@
+//==================================================================================================
+/**
+  EVE - Expressive Vector Engine
+  Copyright 2020 Joel FALCOU
+  Copyright 2020 Jean-Thierry LAPRESTE
+
+  Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+  SPDX-License-Identifier: MIT
+**/
+//==================================================================================================
+#pragma once
+
+#include <eve/traits/element_type.hpp>
+#include <eve/traits/as_wide.hpp>
+#include <eve/forward.hpp>
+#include <concepts>
+
+namespace eve
+{
+  namespace detail
+  {
+    template<typename Wide, typename... ABI>
+    struct  is_abi_compatible
```

Comment:
This should probably be renamed as well?

Reply:
Sure